src/cryptoadvance/specter/templates/includes/qr-scanner.html

```diff
@@ -104,7 +104,12 @@
         // expected: crypto-psbt
         let re3 = /UR:CRYPTO-(.*)/;
         let p3 = result.match(re3);
-        if(p3){
+
+        // check if it is a bcur2 Foundation Passport stuff
+        let re4 = /UR:BYTES\/(.*-*\/)/;
```

Comment:
See here for an example of a UR2 code with a single QR: https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2020-005-ur.md#ur-encoding

Reply:
We need to keep supporting bc-ur-1 because of the Cobo vault.
So this check should be expanded. I'd say if we see UR:BYTES we need to try bcur2 decoding first, and if it fails - treat it as bc-ur-1.